### ISSUE TITLE
Table followed by a list does not parse correctly

### ISSUE THREAD

**Issue by iseulde** [showdownjs/showdown#443]
Tables won't render if it is followed by two line breaks and a list:

```
| Tables        | Are           | Cool  |
| ------------- |:-------------:| -----:|
| **col 3 is**  | right-aligned | $1600 |
| col 2 is      | *centered*    |   $12 |
| zebra stripes | ~~are neat~~  |    $1 |

1. test
```

Three line breaks in between works fine:

```
| Tables        | Are           | Cool  |
| ------------- |:-------------:| -----:|
| **col 3 is**  | right-aligned | $1600 |
| col 2 is      | *centered*    |   $12 |
| zebra stripes | ~~are neat~~  |    $1 |


1. test
```

**Comment by tivie** [comment]
Verified. Bug introduced in version 1.4.4

**Comment by iseulde** [comment]
Thanks!

**Comment by tivie** [comment]
Should be fixed in version 1.7.6

---

>As you know, ShowdownJS is a free library and it will remain free forever. However, maintaining and improving the library costs time and money. Currently, we're looking to improve showdown with automated tests in all browsers and a proper domain and webpage. 500$ should be enough to to keep showdown testing framework running for a year or two.

**If you like our work and find our library useful, please donate [through Pledgie](https://pledgie.com/campaigns/34908) or directly [through paypal](https://www.paypal.me/tiviesantos)!! Your contribution will be greatly appreciated.**
